install/kubernetes/cilium/templates/validate.yaml

```
@@ -30,6 +30,10 @@
   {{- end }}
 {{- end }}
 
+{{- if and .Values.hubble.redact.http.headers.allow .Values.hubble.redact.http.headers.deny }}
+  {{ fail "Hubble allows only one of .Values.hubble.redact.http.headers.allow, .Values.hubble.redact.http.headers.deny to be specified"}}
```

nit: use more direct wording
```suggestion
  {{ fail "Only one of .Values.hubble.redact.http.headers.allow, .Values.hubble.redact.http.headers.deny can be specified"}}
```